app/move/controllers/update/move-date.js

```diff
@@ -1,12 +1,14 @@
-const { get, pick } = require('lodash')
-
-const moveService = require('../../../../common/services/move')
 const { formatDate } = require('../../../../config/nunjucks/filters')
 const CreateMoveDateController = require('../create/move-date')
 
 const UpdateBase = require('./base')
 
 class UpdateMoveDetailsController extends UpdateBase {
+  constructor(options) {
+    super(options)
+    this.saveFields = ['date']
```

Comment:
Could this be derived from `req.form.options.fields`? I think it only includes fields that are available on this step.

Reply:
Unfortunately it includes additional fields (`date_type` and `date_custom`) which would cause this to fail. But come the sooper-dooper model-derived move formatter/unformatter, I think this will be able to be dropped